pkg/transforms/genericResourceConfig.go

```diff
@@ -113,20 +121,28 @@ var defaultTransformConfig = map[string]ResourceConfig{
 	},
 	"VirtualMachineInstance.kubevirt.io": {
 		properties: []ExtractProperty{
+			{Name: "cpu", JSONPath: `{.spec.domain.cpu.cores}`},
 			{Name: "ipaddress", JSONPath: `{.status.interfaces[0].ipAddress}`},
 			{Name: "liveMigratable", JSONPath: `{.status.conditions[?(@.type=='LiveMigratable')].status}`},
+			{Name: "memory", JSONPath: `{.spec.domain.memory.guest}`, DataType: DataTypeBytes},
```

Comment:
@zlayne Can you confirm that adding cpu and memory to `VirtualMachineInstance` won't create a problem on the VM List page when you merge it with the VirtualMachine resource, which has the same keys.

I assume both values should always match, but I don't know if it's guaranteed.

Reply:
There won't be any page break issues if CPU & memory are added to the VMI.

Currently, the merged object will have the CPU & memory of whatever resource was parsed last between the VM & VMI. So if we confirm that those values are always the same, then no changes would be necessary. 

If, however, these values can differ between the VM & VMI then I can add some logic in the frontend to only honor the CPU & Memory from the VMI.